crates/ide-completion/src/completions/postfix.rs

```diff
@@ -84,6 +89,17 @@ pub(crate) fn complete_postfix(
                 )
                 .add_to(acc, ctx.db);
 
+                postfix_snippet(
+                    "lete",
+                    "let Ok else {}",
+                    &if in_loop {
+                        format!("let Ok($1) = {receiver_text} else {{\n    ${{2|continue,break,return|}};\n}};\n$0")
+                    } else {
+                        format!("let Ok($1) = {receiver_text} else {{\n    return;\n}};\n$0")
```

Comment:
We should make the `return` here also a snippet I think

Reply:
Good idea